pallets/multisig/src/lib.rs

```diff
@@ -248,8 +248,7 @@ decl_module! {
 
             // Kill pending proposals if the transaction version is upgraded
             let current_version = <T::Version as Get<RuntimeVersion>>::get().transaction_version;
-            let last_version = TransactionVersion::get();
-            if last_version < current_version {
+            if TransactionVersion::get() < current_version {
                 TransactionVersion::set(current_version);
                 for item in &["Proposals", "ProposalIds", "ProposalDetail", "Votes"] {
                     kill_item(NAME, item.as_bytes())
```

Comment:
We might have to do this in `pips` and `committee` pallets too since they store a `Call` value that could be invalidated by a change to the transaction version.

Reply:
This is by design from prior discussions around Pips.